src/usr/share/netcheck/netcheck.py

```diff
@@ -136,6 +149,9 @@ def start(self):
         # Connect back to connections in priority order.
         self._initial_activate_and_check_connections_in_priority_order(start_time)
 
+        # Check for changes in the default gateway and issue a broadcast if there are any.
```

Comment:
When I suggested you put a comment here, I didn't want you to explain what this does, but why are you doing this in the 'start' method.